docs/ide_integration/r_studio.md

```diff
@@ -0,0 +1,44 @@
+# Developing R scripts in RStudio
+
+You can use `pixi` to manage your R dependencies. The conda-forge channel contains a wide range of R packages that can be installed using `pixi`.
+
+## Installing R packages
+
+R packages are usually prefixed with `r-` in the conda-forge channel. To install an R package, you can use the following command:
+
+```bash
+pixi add r-<package-name>
+# for example
+pixi add r-ggplot2
+```
+
+## Using R packages in RStudio
+
+To use the R packages installed by `pixi` in RStudio, you need to run `rstudio` from an activated environment. This can be achieved by running RStudio from `pixi shell` or from a task in the `pixi.toml` file.
+
+## Full example
+
+Here is an example of a `pixi.toml` file that sets up an RStudio task:
+
+```toml
+[project]
+name = "r"
+channels = ["conda-forge"]
+platforms = ["linux-64", "osx-64", "osx-arm64"]
+
+[target.linux.tasks]
+rstudio = "rstudio"
+
```

Comment:
```suggestion
```
This doesn't really make sense anymore. As that would be the same as `pixi run rstudio` without the task definition for linux.